### DISCUSSION TITLE
retrieving data from Workers KV

### DISCUSSION THREAD

**Discussion by Roy-Nelson-Tous** [frandiox/vitedge#73] [Q&A]
does vitedge  have a way to retrieve data from workers KV natively without using a third party library?

 if not what is the best way to retrieve data from KV for vitedge

**Comment by frandiox** [comment]
Yes, if you install `miniflare@^1.3.3` as a dev dependency you should have access to [KV API](https://developers.cloudflare.com/workers/runtime-apis/kv) in development.

Please tell me if anything related to that doesn't work.
